public/locales/es/translation.json

```diff
@@ -139,10 +141,14 @@
                 "install": "Los servidores de Epic están teniendo una gran interrupción en este momento. ¡El juego no se puede instalar!",
                 "update": "Los servidores de Epic están teniendo una gran interrupción en este momento. ¡El juego no se puede actualizar!"
             },
-            "title": "Advertencia"
+            "title": "Advertencia",
+            "wine-change": {
+                "message": "No hemos podido encontrar la versión de vino seleccionada para ejecutar este título ({{selectedWine}}). {{newline}} Hemos encontrado otra, ¿deseas continuar utilizando {{foundWine}}?"
```

Comment:
This line is translating Wine as the alcoholic drink haha. Should I push a commit here? @BrettCleary

Reply:
this is the only error I seen in the translations to Spanish